tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -28,71 +29,41 @@ public LoginController() {
     }
 
     @Override
-    public HttpResponse run(HttpRequest request) {
-        if (request.isBodyEmpty()) {
-            return redirectLoginPage();
-        }
+    public ForwardResult execute(HttpRequest request, Manager manager) {
         String body = request.getBody();
         Map<String, String> parsedBody = parseBody(body);
 
         String account = parsedBody.get(ACCOUNT_KEY);
         String password = parsedBody.get(PASSWORD_KEY);
 
         Optional<User> optionalUser = userRepository.findByAccount(account);
+        ResponseHeader header = new ResponseHeader();
 
         if (optionalUser.isPresent() && optionalUser.get().checkPassword(password)) {
             User user = optionalUser.get();
-            ResponseHeader header = new ResponseHeader();
-            Manager manager = SessionManager.getInstance();
-
             addSession(request, manager, user, header);
-            return redirectDefaultPage(header);
+            return new ForwardResult(HttpStatusCode.OK, "index.html", header);
```

Comment:
해당 부분은 FOUND로 수정했습니다.
```suggestion
            return new ForwardResult(HttpStatusCode.FOUND, "index.html", header);
```

Reply:
```suggestion
            return new ForwardResult("index.html", header);
```

클래스명에 forward의 의미를 담고 있는데 외부에서 상태 코드를 주입해줄 필요가 있나요? 클라이언트가 다른 상태 코드를 주입하면 클래스명과 다른 동작을 하겠군요?